basket/news/tests/api/test_newsletters.py

```diff
@@ -12,11 +12,13 @@
 @pytest.mark.django_db
 class TestNewslettersAPI:
     def setup_method(self, method):
-        cache.clear()
         self.url = reverse("api.v1:news.newsletters")
         self.n1 = self._add_newsletter("test-1", show=True, order=1)
         self.n2 = self._add_newsletter("test-2", order=2)
 
+    def teardown_method(self, method):
+        cache.clear()
```

Comment:
Worth keeping the cache.clear() in the setup too, in case another test poisons it?